lib/builders/MaterialBuilder.ts

```diff
@@ -11,18 +11,17 @@ import {DefaultConfig} from '../common/DefaultConfig';
 export class MaterialBuilder {
 
   protected _themeBuilder: ThemeBuilder;
+  protected _themes: MdTheme[];
   protected _outputBase: string;
 
   constructor(protected _options: MaterialToolsOptions) {
+    this._outputBase = path.join(this._options.destination, this._options.destinationFilename);
+
     if (this._options.theme || this._options.themes) {
-      let themes = []
+      this._themes = []

```

Comment:
Actually, it might make better sense not to store this on the object at all and move the `let themes = ...` from before to the `buildTheme` method.